sdk/identity/Azure.Identity/src/TokenCredentialOptions.cs

```diff
@@ -29,10 +37,8 @@ public Uri AuthorityHost
         public bool AllowMultiTenantAuthentication { get; set; }
 
         /// <summary>
-        /// Gets or sets value indicating if ETW logging that contains PII content should be logged.
-        /// Setting this property will not disable redaction of <see cref="Request"/> Content. To enable logging of sensitive <see cref="Request.Content"/>
-        /// the <see cref="DiagnosticsOptions.IsLoggingContentEnabled"/> property must be set to <c>true</c>.
+        /// Gets the credential diagnostic options.
         /// </summary>
-        public bool IsLoggingPIIEnabled { get; set; }
+        public new TokenCredentialDiagnosticsOptions Diagnostics => base.Diagnostics as TokenCredentialDiagnosticsOptions;
```

Comment:
Did we decide not to require fancy casting to expose the property?

Reply:
We could require a cast, it would certainly lesson the discoverability of the extended diagnostic features. In the end I  was thinking these don't really need to be hidden. But if others disagree we could not change the return type and simply set it to an instance of `TokenCredentialDiagnosticOptions` and make advanced users who want to enable the extended logging cast it.